### PR TITLE
[CFP-407] FormBuilder migration - Interim fees

### PR DESCRIPTION
**Pull request by naseberry**
#### What

Migrate external_users form step: **interim_fees** form

#### Ticket

[FormBuilder migration - Interim fees](https://dsdmoj.atlassian.net/browse/CFP-407)

#### Why

- Update the service to use GOV.UK styles, components and patterns
- To improve and resolve the service's accessibility issues
- To reduce code complexity
- To improve maintainability

--------

#### TODO (wip)

 - [ ] Review views
 - [ ] Review validators
 - [ ] Review locales
 - [ ] Review specs
 - [ ] Aob

NOTE: If `Interm fees` ticket actioned before `Disbursements fees` ticket, then `Disbursements fees` can be completed at the same time.

### CODE REVIEW

File: app/validators/base_validator.rb [hunk 1]
```diff
@@ -192,6 +192,7 @@ def validate_presence_and_numericality(field, minimum: 0, allow_blank: false)
   def validate_presence_and_numericality_govuk_formbuilder(field, minimum: 0, allow_blank: false)
     validate_presence(field, :blank) unless allow_blank
     validate_float_numericality(field, :numericality, minimum, nil)
+    # validate_amount_less_than_item_max(field)
```
Comment (imtiazAhmd) [inline app/validators/base_validator.rb]: Do we need this commented out line ?

File: config/locales/en/models/claim.yml [hunk 2]
```diff
@@ -262,6 +284,25 @@ en:
...
+            amount:
+              blank: Enter an amount for the interim fee
+              item_max_amount: The amount for the interim fee exceeds the limit
+              numericality: Enter a valid amount for this fee
+              present: Do not enter an amount for the interim fee
+            fee_type_id:
+              blank: Choose a type for the interim fee
+            quantity:
+              blank: Enter a PPE quantity for the interim fee
+              numericality: Enter a valid PPE quantity for the interim fee
```
Comment (imtiazAhmd) [inline config/locales/en/models/claim.yml:321]: Does this `quantity` need `item_max_amount` key?

Comment (naseberry) [reply config/locales/en/models/claim.yml:321]: `item_max_amount` key added

File: config/locales/en/models/claim.yml [hunk 3]
```diff
@@ -286,5 +331,22 @@ en:
               invalid: Enter a valid quantity for the miscellaneous fee
             rate:
               invalid: Enter a valid rate for the miscellaneous fee
+        fee/warrant_fee:
+          attributes:
```
Comment (imtiazAhmd) [inline config/locales/en/models/claim.yml:335]: Missing attributes 'blank'

**Pull request closed without merging**